README_human.md

```diff
@@ -11,15 +11,25 @@ This will usually mean opening a terminal and using `ssh` to connect. The genera
 ```bash
 	ssh username@my.hpc.server.edu.au
 ```
+## Adding Files into a HPC terminal
+To begin, drag the required files into a new terminal. A new terminal should appear on the screen.
```

Comment:
This line is maybe a bit specific and might not be understood by everyone.  A more generic statement would be that below.  

To begin, navigate to the directory containing the files you want to upload to the server.